js/index.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+	//waits for document to fully load to start function
+	$('form').submit(addCity);
+
+	function addCity (event) {
+		event.preventDefault();
+
+		var varcity = $('#city-type').val();
+
+		if (varcity === "NYC" || varcity === "New York City" ||varcity === "New York") {
+			
+			$('body').addClass('nyc');
+			console.log('nyc');
+			//changes background to nyc pic
+		}
+
+		if (varcity === "SF" || varcity === "San Francisco" || varcity === "Bay Area") {
+			
+			$('body').addClass('sf');
+			console.log('sf');
+			// changes background to sanfran pic
+		}
+		if (varcity === "LA" || varcity === "Los Angeles" || varcity === "LAX") {
+			
+			$('body').addClass('la');
```

Comment:
I'll slack you separately about only using `addClass()` in each condition, rather than removing any classes prior to adding the class.